source/manual/load-test-email-alert-api.html.md

```diff
@@ -1,73 +1,27 @@
 ---
 owner_slack: "#govuk-platform-health"
-title: Load Test Email Alert API
+title: Load test Email Alert API
 parent: "/manual.html"
 layout: manual_layout
 section: Emails
 ---
 
-You may wish to load test email-alert-api to get a realistic idea of how the system performs under high load, or if you're making changes to how emails are processed to ensure your changes have the desired effect.
+You may wish to load test Email Alert API to get a realistic idea of how the
+system performs when it has a large quantity of emails to create and send.
+This can be useful to provide data on where the system may have performance
+bottlenecks.
 
-Before you begin, you should post in the #2nd-line channel that you’re about to do some load testing and that as part of that you’ll be disabling puppet runs on Staging for email-alert-api.
+To perform a load test you will need:
 
-> Rake tasks for load testing can be found [here](https://github.com/alphagov/email-alert-api/blob/master/lib/tasks/load_testing.rake).
+- A mechanism to artificially create a quantity of work for Email Alert API to
+  do - we previously had a number of [rake tasks][] to allow this;
+- An approach to simulate the delay of an actual request to Notify - we
+  previously had a [DelayProvider][] class for this.
```

Comment:
```suggestion
  previously used `Kernel.sleep 0.1` to simulate this.
```

I think it's more confusing to link to code that no longer exists, as the reader would have to work out what replaced it.

Reply:
Thanks, I've applied some very similar.